embedding-reports/blazor-reporting.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Blazor Reporting
+page_title: Blazor Reporting – Telerik Reporting
+description: "Blazor Reporting is a framework for generating and delivering dynamic reports in web apps using Blazor technology."
+slug: blazor-reporting
+tags: telerik, reporting, reports, web, blazor
+published: True
+position: 7
+---
+
+# Embedding Telerik Reporting in Blazor Reporting Applications
+
+[Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor) is the latest Microsoft Web technology letting you build modern cross-platform client-side, server-side, and hybrid applications relying mainly on C# code for the web site functionality. This article offers an overview of the most popular approaches to integrate Telerik Reporting in Blazor Web Reporting applications.
+
+## Supported Operating Systems and Docker Containers
+
+The article [Available .NET Support for Operating Systems and Docker Containers]({%slug telerikreporting/using-reports-in-applications/dot-net-core-support%}) elaborates on the specifics when deploying Telerik Reporting on different environments.
+
+## Displaying Reports in Blazor Web Reporting Applications
```

Comment:
My assumption is that displaying the report is one of the later steps. Don't you need to first design the report or integrate it into the app?
Consider moving this section further down and placing here the section that corresponds to the most logical first step in the process.